src/main/java/com/aws/greengrass/shadowmanager/util/Validator.java

```diff
@@ -150,4 +148,15 @@ public static void validateLocalShadowRequestsPerThingPerSecond(int maxLocalShad
         }
     }
 
+    /**
+     * Validate that a shadow size does not exceed the maximum.
+     * @param size size of shadow
+     * @throws InvalidRequestParametersException if the shadow is too large
+     */
+    public static void validateShadowSize(int size) throws InvalidRequestParametersException {
+        if (size > Validator.getMaxShadowDocumentSize()) {
```

Comment:
nit: why are we using the getter here and not the variable itself?

Reply:
copy/paste error